tests/integration/test_integration.py

```diff
@@ -5,12 +5,12 @@
 import logging
 import time
 from pathlib import Path
-from typing import Tuple
+from typing import Optional, Tuple
 
 import pytest
 import requests.exceptions  # type: ignore[import]
```

Comment:
You could add `types-requests` to the deps of the `static` env in the `tox.ini` and avoid this ignore.

Reply:
Done.